### ISSUE TITLE
Error when emptying recycle bin

### ISSUE THREAD

**Issue by maggie0002** [BookStackApp/BookStack#2556]
**Describe the bug**
When emptying the recycle bin (or trying to delete certain items individually) I am getting an error on a number of entries. It is not on all items, just a set few. I can restore them, but can't delete them. 

I recently switched to S3 storage, and wondered if these were entries from before the move. I switched back to local storage but still getting the issue. 

```
Symfony\Component\Debug\Exception\FatalThrowableError
Return value of BookStack\Entities\Tools\TrashCan::destroyEntity() must be of the type int, none returned
```

**Your Configuration (please complete the following information):**
Linuxserver docker image
BookStack v0.31.3

**Comment by maggie0002** [comment]
I also changed my email address and name of the admin account from the default admin@admin.com to my own email. I can be sure, but it's possible all the shelves/books/pages that are causing the error were made with this account, and it would have been before I change the email.

**Comment by ssddanbrown** [comment]
Hi @maggie0002,

This was fixed as part of v0.31.6: https://github.com/BookStackApp/BookStack/releases/tag/v0.31.6
I'll therefore close off this issue but if the issue remains after updating feel free to open an new issue.
